desktop/app/app-state.js

```diff
@@ -0,0 +1,195 @@
+//
```

Comment:
Added flow types...

BTW, I have to manually use a custom flow config cause Atom editor doesn't support a flowconfig from the shared. See https://keybase.atlassian.net/browse/DESKTOP-1486